src/editor/codemirror/CodeMirror.tsx

```diff
@@ -122,11 +124,20 @@ const CodeMirror = ({
           // Extensions we enable/disable based on props.
           compartment.of([
             client
-              ? languageServer(client, uri, intl, logging, {
-                  signatureHelp: {
-                    automatic: parameterHelpOption === "automatic",
-                  },
-                })
+              ? languageServer(
+                  client,
+                  uri,
+                  intl,
+                  logging,
+                  apiReferenceMap.status === "ok"
+                    ? apiReferenceMap.content
+                    : {},
```

Comment:
If the data fails to load, the `Help` button is never displayed. There are no errors, but nothing is logged.

Reply:
@microbit-matt-hillsdon Correction, logging is already handled inside `useContent` in `documentation-hooks.tsx` which we make use of here.